test/foundry/HederaFungibleToken.t.sol

```diff
@@ -234,6 +234,48 @@ contract HederaFungibleTokenTest is HederaTokenUtils, HederaFungibleTokenUtils {
         assertEq(success, true, "expected burn to succeed");
     }
 
+    function test_CanAssociateAndDissociateDirectly() public {
+        IHederaTokenService.TokenKey[] memory keys = new IHederaTokenService.TokenKey[](0);
+        address tokenAddress = _createSimpleMockFungibleToken(alice, keys);
+
+        bool success;
+        uint256 amount = 1e8;
+
+        TransferParams memory transferParams = TransferParams({
```

Comment:
should we change `transferParams` to `transferFromAliceToBob` to keep consistency and also be more explicit with the test?